forge-gui/res/cardsfolder/upcoming/mold_folk.txt

```diff
@@ -0,0 +1,10 @@
+Name:Mold Folk
+ManaCost:1 B
+Types:Creature Fungus Warrior
+PT:1/1
+K:Lifelink
+A:AB$ PutCounter | Cost$ 1 Sac<1/Creature.Other;Artifact/another creature or artifact> | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Mold Harvest — {1}, Sacrifice another creature or an artifact: Put a +1/+1 counter on CARDNAME.
+SVar:AIPreference:SacCost$Artifact.Token,Creature.Other+cmcLE2,Artifact.cmcEQ1
+SVar:BuffedBy:Artifact,Creature
```

Comment:
I don't think line is appropriate for this card... maybe this would be better
`DeckHints:Type$Artifact` since most decks have creatures?